### ISSUE TITLE
[blocked by spec errors] [Google Map Support] Edit feature button no longer works after switching from Google Maps basemap to Leaflet basemap (or viceversa)

### ISSUE THREAD

**Issue by alonsogarciapablo** [CartoDB/cartodb#11474]
**STR:**

- Open a map.
- Click on a feature and click the edit button.
- Editing should work like a charm.
- Change the basemap (Leaflet -> Google Maps or viceversa).
- Click on a feature again. You should see the edit button.
- Click the edit button.

**Expected results:**

- You should be able to edit the feature, as you did before.

**Actual results:**

- Edit feature overlay is closed and feature is not editable.

I triaged this and it looks like, for some reason, [this event handler](https://github.com/CartoDB/cartodb/blob/master/lib/assets/core/javascripts/cartodb3/deep-insights-integration/edit-feature-overlay.js#L160-L184) is no longer invoked after a provider change, so clicking the button has no effect.

**Comment by xavijam** [comment]
Maybe, after https://github.com/CartoDB/cartodb/pull/11448, this is gonna work again, but not sure 100%.

**Comment by matallo** [comment]
did a quick test (and built the dependencies already for this) and when merging both branches I can confirm this is still happening

**Comment by javierarce** [comment]
I'm unassigning myself from this issue, since the CR passed but there are some specs failing [in the PR](https://github.com/CartoDB/cartodb/pull/11494). Once the build turns green, this ticket could be moved to Acceptance.

**Comment by xavijam** [comment]
Taking myself, in order to test it, once more. Then It will be merged into `google-maps` branch.

**Comment by xavijam** [comment]
Merged in `google-maps` branch.
